CONTRIBUTING.md

```diff
@@ -175,23 +175,32 @@ To add a new component, add an annotation to `components` package:
 ```kotlin
 import net.skyscanner.backpack.demo.meta.Component
 
-@Component("My Component")
+@Component("My Component") // name of the component to be used in UI
 annotation class MyComponent
 ```
 
-To add a new demo screen, annotate your composable function with the component and story annotations:
+To add a new demo screen (we call it a story), annotate your composable function with the component and story annotations:
 
 ```kotlin
 @Composable
 @MyComponent
-@ComposeStory
-// or @ComposeStory("Custom story name")
-// or @ViewStory for view components
+@ComposeStory("Name to be used in samples list for component, `Default` is used by default")
```

Comment:
again here this is not just use for the list, but also for the screenshot name. Also worth clarifying that you only really need to specify a name if there's multiple sections to a component. So I'd add a section explaining how that works as well, as i'm not sure it'll be clear to somebody that adding multiple stories will create multiple sections

Reply:
Done